PokeAlarm/Manager.py

```diff
@@ -263,7 +274,10 @@ def handle_pokemon(self, pkmn):
             'move_2_damage': get_move_damage(move_2_id),
             'move_2_dps': get_move_dps(move_2_id),
             'move_2_duration': get_move_duration(move_2_id),
-            'move_2_energy': get_move_energy(move_2_id)
+            'move_2_energy': get_move_energy(move_2_id),
+            'height': "{:.2f}".format(height) if height != 'unkn' else 'unkn',
```

Comment:
I think that `{:.1f}` would be a better fit because it matches the default IVs (and who really cares about weight to that precision anyway?)